src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -728,208 +751,266 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
                 return;
             }
 
-            var elem = self._mediaElement;
+            const elem = this._mediaElement;
             if (!elem) {
                 return;
             }
 
             // https://msdn.microsoft.com/en-us/library/hh772507(v=vs.85).aspx
 
-            var elemAudioTracks = elem.audioTracks || [];
+            const elemAudioTracks = elem.audioTracks || [];
             console.debug('found ' + elemAudioTracks.length + ' audio tracks');
 
-            for (i = 0, length = elemAudioTracks.length; i < length; i++) {
-
+            elemAudioTracks.forEach((audioTrack, i) => {
```

Comment:
```suggestion
            for (const [index, audiotTrack] of elemAudioTracks.entries()) {
```
Should work here (untested)

Reply:
No there'll be one closing parenthesis to much, but anyway is there a specific reason to change that? Or is it just aesthetic?

Reply:
part of #1460